uuid.go

```diff
@@ -97,6 +97,16 @@ func ParseBytes(b []byte) (UUID, error) {
 	return uuid, nil
 }
 
+// MustParse is like Parse but panics if the string cannot be parsed.
+// It simplifies safe initialization of global variables holding compiled UUIDs.
+func MustParse(s string) UUID {
+	uuid, err := Parse(s)
+	if err != nil {
+		panic(`uuid: Parse(` + s + `): ` + err.Error())
+	}
+	return uuid
+}
```

Comment:
Maybe a generic Must function:

func Must(uuid UUID, err error) UUID {
    if err != nil { panic('uuid: ' + err.Error) }
    return uuid
}

And then you can say:

uuid.Must(uuid.Parse(s))
uuid.Must(uuid.FromBytes(b))

and so on.

What do you think?

Reply:
The Must function you mention is already in code.
Indeed `uuid.Must(uuid.Parse(s))` could work just as well.
But I think it would help adding an example for this at least.
Do you want me to rework this PR to instead add a example function for `uuid.Must(uuid.Parse(s))`, or in a new PR?

With this patch I was just trying to follow some convention I imagined was around. It doesnt seem to be very enforced:

```
$ cd ~/dev/go-src
$ rg "func Must"
api/go1.txt
3374:pkg html/template, func Must(*Template, error) *Template
5523:pkg regexp, func MustCompile(string) *Regexp
5524:pkg regexp, func MustCompilePOSIX(string) *Regexp
28611:pkg syscall (windows-386), func MustLoadDLL(string) *DLL
29540:pkg syscall (windows-amd64), func MustLoadDLL(string) *DLL
30338:pkg text/template, func Must(*Template, error) *Template'
...
```